pkg/config/log.go

```diff
@@ -132,6 +134,49 @@ func SetupJMXLogger(logFile, syslogURI string, syslogRFC, logToConsole, jsonForm
 	return nil
 }
 
+// SetupDogstatsdLogger sets up a logger with dogstatsd logger name and log level
+// if a non empty logFile is provided, it will also log to the file
+func SetupDogstatsdLogger(logFile, syslogURI string, syslogRFC, jsonFormat bool) (seelog.LoggerInterface, error) {
+	seelogLogLevel, err := validateLogLevel("info")
+	if err != nil {
+		return nil, err
+	}
+
+	dogstatsdSeelogConfig, err = buildDogstatsdLoggerConfig(DogstatsDLoggerName, seelogLogLevel, logFile, syslogURI, syslogRFC, jsonFormat)
+	if err != nil {
+		return nil, err
+	}
+
+	dogstatsdLoggerInterface, err := GenerateLoggerInterface(dogstatsdSeelogConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	return dogstatsdLoggerInterface, nil
+
+}
+
+func buildDogstatsdLoggerConfig(loggerName LoggerName, seelogLogLevel, logFile, syslogURI string, syslogRFC, jsonFormat bool) (*seelogCfg.Config, error) {
```

Comment:
`syslogURI` is unused and can be removed. 
I'm not sure you really need to expose the other options (`syslogRFC`, `jsonFormat`) since this log file has a specific purpose and is not a general purpose log file. They can probably be hard coded to the desired format.